source/docs/software/commandbased/command-based-changes.rst

```diff
@@ -66,6 +66,9 @@ Changes to Command
 * As noted earlier, ``Command`` is now an interface (`Java <https://first.wpi.edu/FRC/roborio/development/docs/java/edu/wpi/first/wpilibj2/command/Command.html>`__, `C++ <https://first.wpi.edu/FRC/roborio/development/docs/cpp/classfrc2_1_1Command.html>`__); the closest equivalent of the old ``Command`` is the new ``CommandBase`` class.  Many of the Sendable-related constructor overloads have been removed to reduce clutter; users can call the setters directly from their own constructor, if needed.
 * Commands no longer handle their own scheduling state; this is now the responsibility of the scheduler.
 * The ``interrupted()`` method has been rolled into the ``end()`` method, which now takes a parameter specifying whether the command was interrupted (``false`` if it ended normally).
+* The ``requires()`` method has been renamed to ``addRequirement()``.
+* ``void setRunsWhenDisabled(boolean disabled)`` has been replaced by an overrideable ``runsWhenDisabled()`` method.  Commands that should run when disabled should override this method to return true.
+* ''void setInterruptible(boolean interruptible)'' has been removed; interruptibility is no longer an innate property of commands, and can be set when the command is scheduled.
```

Comment:
Should use backticks and not single quotation marks around the `void setInterruptible(...)`.